akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -174,12 +191,17 @@ class ClusterClient(settings: ClusterClientSettings) extends Actor with Stash wi
 
   require(initialContacts.nonEmpty, "initialContacts must be defined")
 
+  // FIXME use new constructor of DeadlineFailureDetector with heartbeatInterval
+  val failureDetector = new DeadlineFailureDetector(acceptableHeartbeatPause)
```

Comment:
note to self: remember to change this before merging, i.e. when #17604 has been merged